uint/src/uint.rs

```diff
@@ -800,13 +800,10 @@ macro_rules! construct_uint {
 			}
 
 			/// The maximum value which can be inhabited by this type.
+			#[deprecated(since = "0.10.0", note = "replaced by the `MAX` associated constant on this type")]
```

Comment:
Sorry about this, but the std lib doesn't deprecate it as of yet, but will in the future it seems.doc
```suggestion
```

Reply:
Oh, they have a _will be deprecated_ state. Strange.